apps/meteor/client/views/admin/featurePreview/AdminFeaturePreviewPage.tsx

```diff
@@ -0,0 +1,172 @@
+import {
+	ButtonGroup,
+	Button,
+	Box,
+	ToggleSwitch,
+	Accordion,
+	Field,
+	FieldGroup,
+	FieldLabel,
+	FieldRow,
+	FieldHint,
+	Callout,
+} from '@rocket.chat/fuselage';
+import type { FeaturePreviewProps } from '@rocket.chat/ui-client';
+import { useDefaultSettingFeaturePreviewList } from '@rocket.chat/ui-client';
+import type { TranslationKey } from '@rocket.chat/ui-contexts';
+import { useToastMessageDispatch, useTranslation, useEndpoint, useSettingsDispatch } from '@rocket.chat/ui-contexts';
+import type { ChangeEvent } from 'react';
+import React, { useEffect, Fragment } from 'react';
+import { useForm } from 'react-hook-form';
+
+import { Page, PageHeader, PageScrollableContentWithShadow, PageFooter } from '../../../components/Page';
+import { useEditableSetting } from '../EditableSettingsContext';
+import Setting from '../settings/Setting';
+import SettingsGroupPageSkeleton from '../settings/SettingsGroupPage/SettingsGroupPageSkeleton';
+
+const handleEnableQuery = (features: FeaturePreviewProps[]) => {
```

Comment:
since it's the same `handleEnableQuery` we use in the accounts page, lets try to abstract into a hook